src/providers/exoscale/mod.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! metadata fetcher for the exoscale provider
```

Comment:
If you have a metadata schema/doc-reference on some public URL, can you please link it here?

Reply:
Yes sure I'll have a look for that 👍

Reply:
ok fixed